opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -895,10 +904,7 @@ namespace Opm {
         } else if (setCount != 0) {
             std::string msg = "BOX modifiers on keywords must be either "
                 "specified completely or not at all. Ignoring.";
-            OpmLog::addMessage(Log::MessageType::Error,
-                               Log::fileMessage(deckKeyword.getFileName(),
-                                                deckKeyword.getLineNumber(),
-                                                msg));
+            m_intGridProperties.getMessageContainer().error(deckKeyword.getFileName() + std::to_string(deckKeyword.getLineNumber()) + msg);
```

Comment:
No, because the ``error()`` function accept a ``std::string`` argument, so  it should be ``+``, not ``,``